docs/cluster_and_device_type_dev/cluster_and_device_type_dev.md

```diff
@@ -172,10 +172,10 @@ persistence
 
 -   When using AttributeAccessInterface, you need to manage any Attributes that
     require Persistence.
--   This can be done by using GetSafeAttributePersistenceProvider()
+-   This can be done by using GetAttributePersistenceProvider()
 -   This provides a useful API for Reading & Writing values of any type to the
```

Comment:
We use https://github.com/project-chip/connectedhomeip/blob/master/src/app/persistence/AttributePersistence.h#L35 for the nicer API